packages/compass-e2e-tests/tsconfig.json

```diff
@@ -1,4 +1,6 @@
 {
   "extends": "@mongodb-js/tsconfig-compass/tsconfig.common.json",
-  "compilerOptions": {}
+  "compilerOptions": {
+    "types": ["node", "@wdio/globals/types"]
```

Comment:
webdriverio's documentation says to do this. not 100% sure that it is absolutely necessary.

Reply:
I think ts parses tsconfig as a json5, so comments are allowed if you want to add a reference to the doc here (no strong preference for doing this though, just FYI)